test/puppetlabs/pcp/messaging_test.clj

```diff
@@ -173,3 +173,13 @@
       (is (= client (client/wait-for-association client (* 40 1000))))
       (is (= false (client/associating? client)))
       (is (= true (client/associated? client))))))
+
+(deftest connect-with-too-small-message-size
+  (with-app-with-config
+    app
+    [authorization-service broker-service jetty9-service webrouting-service metrics-service]
+    broker-config
+    (with-open [client (connect-client-config (assoc (client-config "client01")
+                                                     :max-message-size 128)
+                                              (constantly true))]
+      (is (= nil (client/wait-for-association client 1000))))))
```

Comment:
This looks like we should check that the following message was logged:
```
2016-07-13 19:33:30,187 DEBUG [p.p.client] WebSocket closed 1009 Binary message size [289] exceeds maximum size [128]
```
using the `(with-test-logging )` and `(logged? )` utils, like we do for instance here:
https://github.com/puppetlabs/clj-pxp-puppet/blob/master/test/integration/puppetlabs/pxp/puppet_run_test.clj#L257
and here:
https://github.com/puppetlabs/clj-pxp-puppet/blob/master/test/integration/puppetlabs/pxp/puppet_run_test.clj#L268